app/assets/stylesheets/components/_select-with-search.scss

```diff
@@ -0,0 +1,415 @@
+      height: 100%;
+      line-height: 1;
+      opacity: .75;
+      border-radius: 0;
+
+      &:hover,
+      &:focus {
+        opacity: 1;
+      }
+    }
+  }
+
+  .choices__inner {
+    display: inline-block;
+    vertical-align: top;
+    width: 100%;
+    background-color: $background;
+    padding: 5px;
+    border: 2px solid #0b0c0c;
+    font-size: 19px;
+    min-height: 44px;
+    overflow: hidden;
+  }
+
+  &.govuk-form-group--error .choices:not(.is-active, .is-focused, .is-open) .choices__inner {
+    border-color: $govuk-error-colour;
+  }
+
+  .choices.is-focused .choices__inner,
+  .choices.is-open .choices__inner {
+    outline: $focus-outline;
+    outline-offset: 0;
+    box-shadow: inset 0 0 0 2px;
+  }
+
+  .choices__list {
+    margin: 0;
+    padding-left: 0;
+    list-style: none;
+  }
+
+  .choices__list--single {
+    display: inline-block;
+    padding: 4px 16px 4px 4px;
+    width: 100%;
+  }
+
+  [dir="rtl"] .choices__list--single {
+    padding-right: 4px;
+    padding-left: 16px;
+  }
+
+  .choices__list--single .choices__item {
+    width: 100%;
+  }
+
+  .choices__list--multiple {
+    display: inline;
+  }
+
+  .choices__list--multiple .choices__item {
+    display: inline-block;
+    vertical-align: middle;
+    padding: 4px 10px;
+    margin-right: 4px;
+    margin-bottom: 4px;
+    background-color: #f3f2f1;
+    box-shadow: 0 2px 0 #929191;
+    line-height: 1;
+    color: #0b0c0c;
+    word-break: break-all;
+    box-sizing: border-box;
+
+    &[data-deletable] {
+      padding-right: 5px;
+    }
+
+    [dir="rtl"] & {
+      margin-right: 0;
+      margin-left: 3.75px;
+    }
+
+    &.is-highlighted {
+      background-color: $govuk-focus-colour;
+    }
+
+    .is-disabled & {
+      background-color: #aaaaaa;
+      border: 1px solid #919191;
+    }
+  }
+
+  // Dropdown
+  .choices__list--dropdown,
+  .choices__list[aria-expanded] {
+    visibility: hidden;
+    z-index: 2;
+    position: absolute;
+    width: 100%;
+    background-color: $background;
+    border: 2px solid #0b0c0c;
+    border-top-width: 0;
+    top: 100%;
+    overflow: hidden;
+    word-break: break-all;
+    will-change: visibility;
+
+    &.is-active {
+      visibility: visible;
+    }
+
+    .is-flipped & {
+      top: auto;
+      bottom: 100%;
+      margin-top: 0;
+      margin-bottom: -1px;
+      border-top-width: 2px;
+      border-bottom-width: 0;
+    }
+  }
+
+  .choices__list--dropdown .choices__list,
+  .choices__list[aria-expanded] .choices__list {
+    position: relative;
+    max-height: 310px;
+    overflow: auto;
+    -webkit-overflow-scrolling: touch;
+    will-change: scroll-position;
+  }
+
+  .choices__list--dropdown .choices__item,
+  .choices__list[aria-expanded] .choices__item {
+    position: relative;
+    padding: 10px;
+    font-size: 19px;
+    border-bottom: 1px solid #b1b4b6;
+
+    &:last-child {
+      border-bottom: 0;
+    }
+
+    &:nth-of-type(even) {
+      background-color: #fafafa;
```

Comment:
```suggestion
      background-color: govuk-colour("light-grey");
```

I think we should update this colour with something with a bit more contrast.


## Before
<img width="1335" alt="Screenshot 2023-03-27 at 4 38 10 pm" src="https://user-images.githubusercontent.com/4599889/227992649-b2128869-f1e3-45f6-988f-b5f44208e9a5.png">

## After

<img width="1245" alt="Screenshot 2023-03-27 at 4 40 56 pm" src="https://user-images.githubusercontent.com/4599889/227992738-babd8720-9226-4a83-bed8-dd3ec23da978.png">

Reply:
Thanks @DilwoarH 🎉 

I've tried as much as possible to re-use the existing styles established by [Accessible Autocomplete](https://alphagov.github.io/accessible-autocomplete/).

That includes the alternating colours of options, which Accessible Autocomplete defines as:

```css
.autocomplete__option--odd {
  background-color: #FAFAFA;
}
```

([view source](https://github.com/alphagov/accessible-autocomplete/blob/a79d8f7c07e6e1af296ee0f913447a248faa550d/src/autocomplete.css#L104-L106))

So that's why I've gone with `#FAFAFA` in this component, too.